tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Course_Information_f909e0.yaml

```diff
@@ -178,6 +264,6 @@ query_context:
   result_format: json
   result_type: full
 slice_name: Course Information
-uuid: fa249dda-78da-4ccc-9ef3-39177e6aae0c
+uuid: f909e03a-06a9-4d04-9ea0-243a576093d4
```

Comment:
Changing those assets UUIDs can cause some problems. Please revert this change and all the references to this new UUID